client/src/utils/Notification.tsx

```diff
@@ -0,0 +1,29 @@
+import { notification } from 'antd';
+
+export const sendSuccessNotification = (message?: string, description?: string): void => {
```

Comment:
Should we not ensure that at least one of these parameters is present?

Reply:
Title and description is not required, as we have discussed.
Notification will be displayed on the basis of their type.

Reply:
> Title and description is not required, as we have discussed.

Yes we have.  Do we anticipate a situation where we will send a notification that has no title and no description?  Why would we do that?

Reply:
So should I make title mandatory ?